File: src/A Different Problem/different.py
# Trying sys, from Kattis's documentation
# https://open.kattis.com/help/python3

import sys
 
for line in sys.stdin:
    ab = line.split()
    a = int(ab[0])
    b = int(ab[1])
    print(abs(a-b))

File: src/A Different Problem/different.kt
// https://open.kattis.com/help/kotlin

fun main(args: Array<String>) {
    var line = readLine();
    while (line != null) {
        var (a, b) = line.split(' ');
        println(Math.abs(a.toLong() - b.toLong()));
        line = readLine();
    }
}